files/uzblmonitor

```diff
@@ -52,6 +44,16 @@ class Browser(object):
             self.sock.close()
             self.sock = None
 
+    def read_config(self):
+        config = ConfigParser.RawConfigParser()
+        config.read(CONFIG_FILE)
+        return config
+
+    def reload_config(self):
```

Comment:
We could just get rid of supporting reload and just let upstart restart it. That's how we do it already; might as well make it official.